hydra-cluster/src/Hydra/Cluster/Scenarios.hs

```diff
@@ -26,12 +26,18 @@ import Hydra.Party (Party)
 import HydraNode (EndToEndLog (..), input, output, send, waitFor, waitForAllMatch, waitMatch, withHydraNode)
 import Test.Hspec.Expectations (shouldBe)
 
+-- | Determine the amount we want to grab from the faucet based on network
+seedFromFaucetAmount :: NetworkId -> Lovelace
+seedFromFaucetAmount Mainnet = 100_000_000
+seedFromFaucetAmount _ = 100_000_000
```

Comment:
Could: not do this, but just find a common denominator between networks.

I see why you want to do this, but I think it's not important how much we seed things with. It just needs to meet the minimum UTxO requirements. We don't expect these tests don't behave different with big or small amounts (if we would, we also should test NFTs). These are only E2E or even smoke tests - their scope is limited.

This parameter can stay uniform for all networks as we do not change valuation or minima in the parameters.